style.css

```diff
@@ -0,0 +1,818 @@
+body,
+h1,
+h2,
+h3,
+h4,
+p { margin: 0; }
+
+body {
+  min-height: 100vh;
+  line-height: 1.5;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+a { text-decoration: none; }
+
+/* Desktop */
+
+/* Шапка */
+.header,
+.header__logo,
+.header__menu,
+.header__wrapper { display: flex; }
+
+.header,
+.header__menu,
+.header__wrapper_center { justify-content: space-between; }
+
+.header__logo,
+.header__wrapper_right { justify-content: center; }
+
+.header {
+  align-items: center;
+  height: 104px;
+  margin: 0 auto;
+  margin-bottom: 30px;
+}
+
+.header__logo {
+  min-width: 167px;
+  flex: .18;
+}
+
+.header__menu {
+  height: 24px;
+  flex: .675;
+}
+
+.header__wrapper_center {
+  min-width: 360px;
+  padding-left: 54px;
+  flex: .353;
+}
+
+.header__link,
+.header__burger { padding-top: 2px; }
+
+.header__link { color: #000; }
+
+.header__wrapper_right {
+  min-width: 215px;
+  flex: .232;
+}
+
+.header__link_right { margin-right: 30px; }
+
+.header__burger:hover { cursor: pointer; }
+
+.burger__icon {
+  display: block;
+  width: 25px;
+  height: 2.5px;
+  margin-bottom: 4px;
+  background-color: #e51937;
+  border-radius: 0 0 5px 5px;
+}
+
+.header__link_center:first-child {
+  border-bottom: 2px solid #e51937;
+  border-radius: 2.5px;
+  color: #e51937;
+}
+
+.header__link:hover,
+.header__burger:hover { padding-top: 0; }
+
+/* Первый экран */
+.section-main {
+  margin-bottom: 34px;
+  background: url(./Project/tesla-wordmark-red_big.svg) no-repeat;
+  background-position: top;
+}
+
+.section-main__title,
+.section-main__subtitle { text-align: center; }
+
+.section-main__title { height: 101px; }
+
+.section-main__subtitle,
+.section-main__img { margin-bottom: 26px; }
+
+.section-main__modelS-logo { width: 95%; }
+
+.section-main__car-logo,
+.section-main__wrapper { margin: auto; }
+
+.section-main__wrapper,
+.section-main__item-info,
+.section-main__block-buttons,
+.section-main__character,
+.buttons,
+.button,
+.button-order,
+.button-order__link,
+.section-main__control-down { display: flex; }
+
+.section-main__wrapper,
+.section-main__item-info { justify-content: space-between; }
+
+.section-main__wrapper {
+  min-width: 700px;
+  width: 56%;
+  margin-bottom: 32px;
+}
+
+.section-main__item-info { flex: .48; }
+
+.character_right { padding-right: 5px; }
+
+.character__icon,
+.button__icon { margin-right: 10px; }
+
+.section-main__block-buttons,
+.button-order__link,
+.button,
+.button-order { align-items: center; }
+
+.button__link_test-drive { color: #000; }
+
+.button:hover { cursor: pointer; }
+
+.button:hover,
+.button-order__link:hover { padding-bottom: 2px; }
+
+.section-main__block-buttons { flex: .4; }
+
+.section-main__button { min-width: 121px; }
+
+.section-main__button-order { width: 51%; }
+
+.button-order {
+  height: 38px;
+  padding-left: 34px;
+}
+
+.button-order__link,
+.section-main__control-down { justify-content: center; }
+
+.button-order__link {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".button-order__link" to come before selector ".button-order__link:hover" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)